TwilioVerifySDK/TwilioSecurity/Sources/Biometrics/BiometricError.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  BiometricError.swift
+//  TwilioVerifySDK
+//
+//  Created by Alejandro Orozco Builes on 17/09/21.
+//  Copyright © 2021 Twilio. All rights reserved.
+//
+
+import Foundation
+import LocalAuthentication
+
+public enum BiometricError: Error, LocalizedError {
+
+    // MARK: - Biometric LAError cases
+
+    case appCancel
+    case authenticationFailed
+    case invalidContext
+    case notInteractive
+    case passcodeNotSet
+    case systemCancel
+    case userCancel
+    case userFallback
+    case biometryLockout
+    case biometryNotAvailable
+    case biometryNotEnrolled
+
+    public var errorDescription: String? {
+        return "Biometric error: LAError \(String(describing: self))"
+    }
+
+    // MARK: - Resolution
+
+    public static func given(_ error: NSError?) -> Self? {
+      guard let error = error else {
+        return nil
+      }
+
+      switch LAError.Code(rawValue: error.code) {
+        case .appCancel: return .appCancel
```

Comment:
What is the difference between the cancels: app, user and system?

Reply:
The description is written in the LAError doc's. Maybe we can add just the link in the enums description https://developer.apple.com/documentation/localauthentication/laerror, so people can read the description from Apples doc? 
```swift
/// Authentication was canceled by application (e.g. invalidate was called while
/// authentication was in progress).
@available(iOS 9.0, *)
case appCancel = -9

// IE
authenticator.context.invalidate()

____________________________________

/// Authentication was canceled by user (e.g. tapped Cancel button).
public static var userCancel: LAError.Code { get }

____________________________________

/// Authentication was canceled by system (e.g. another application went to foreground).
public static var systemCancel: LAError.Code { get }

____________________________________
```